presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -115,11 +115,21 @@ public void deployDefaultConfig(Path configDir, Path dataDir, Path pluginDir)
                 // TODO: Include node.id in install requests
                 updateProperty(configDir.resolve("node.properties"),
                         "node.id", randomUUID().toString());
+                /**
+                 * To prevent conflicts between Presto-Manager configs and
+                 * Presto configs present in {@link #defaultConfig} directory
+                 */
+                Path nodePropertiesPath = configDir.toAbsolutePath().resolve("node.properties");
```

Comment:
So the following three files are needed for Presto to start:
`jvm.config`, `config.properties`, `node.properties`
Most of the configurable stuff in `node.properties` can be obtained from presto-manager configuration.
But for `jvm.config` & `config.properties` need to be specified by the user.
I think this can be upgraded in the future as an enhancement